### PR TITLE
Add test for doc comment attached to `%%private`

### PR DESCRIPTION
**Pull request by anmonteiro**
TODO from #535

### CODE REVIEW

File: test/melange-ppx-private.t [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (using melange 0.1)
+  > EOF
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
+  >  (alias mel)
+  >  (emit_stdlib false)
+  >  ;(preprocess (pps melange.ppx))
```
Comment (jchavarri) [inline test/melange-ppx-private.t:13]: I don't get this change. If the ppx is disabled, then we don't see the error, right?

**Review by jchavarri — COMMENTED** [review]
Thanks for recovering this todo, it completely slipped from my mind 😓 

I don't understand why the `preprocess` field is commented out?

File: test/melange-ppx-private.t (hunk 1 above)
Comment (anmonteiro) [reply test/melange-ppx-private.t:13]: direct link to diff: https://github.com/melange-re/melange/commit/d87e473823762fc0901efac86594bd196e477dec#diff-cf3151c38c0d5e126958e640fd71862a0d3ecf6ec26f7ce935b1d3028062772d

Comment (jchavarri) [reply test/melange-ppx-private.t:13]: I see, thanks!

**Pull request merged**